lib/plugins/aws/deploy/lib/updateStack.js

```diff
@@ -6,6 +6,8 @@ const path = require('path');
 
 module.exports = {
   update() {
+    const templateUrl = `https://s3.amazonaws.com/${this.bucketName}/compiled-cloudformation-template.json`;
```

Comment:
Maybe make this a bit more random, or with a timestamp in the file name? This way you could potentially rollback if needed? Otherwise the file would be overwritten.